src/__tests__/calculate.test.jsx

```diff
@@ -0,0 +1,38 @@
+import calculate from '../logic/calculate';
+
+describe('Check calculate function behavior according to the operation', () => {
```

Comment:
- [ ] As asked by the previous reviewer please include snapshots as they are a project requirement
![image](https://user-images.githubusercontent.com/62705947/224022706-66402437-e458-4386-9a2c-ec960fc7aa3c.png)